functional_tests/steps/home_page.py

```diff
@@ -15,3 +15,8 @@ def user_sees_the_beta_homepage(context: Context) -> None:
     expect(context.page.get_by_label("Office for National Statistics homepage")).to_be_visible()
     expect(context.page.get_by_text("This is a new service.")).to_be_visible()
     expect(context.page.get_by_text("Beta")).to_be_visible()
+
+
+@then("they cannot see the breadcrumbs")
+def user_does_not_sees_on_the_page(context: Context) -> None:
+    expect(context.page.get_by_label("Breadcrumbs")).not_to_be_visible()
```

Comment:
I think this step should be moved to a new `functional_tests/steps/breadcrumbs.py` steps file, since it is used by multiple different features.